.gitignore

```diff
@@ -24,3 +24,13 @@ install/windows/*.exe
 /_install/
 /compile_commands.json
 /.ccls-cache/
+
+# CMake
+CMakeFiles
```

Comment:
This encourages in-source buliding. I would recommend people to use `_build/` as a build dir (or just `build` when Premake is removed).

Reply:
thats a personal preference IMHO, i prefer in source builds: this line really doesn't hurt.